File: 백준/Bronze/1546. 평균/평균.py
N = int(input())
tmp = input().split(' ')
M = 0
tot = 0
for n in range(N):
    tmp[n] = int(tmp[n])
    tot += tmp[n]
    if tmp[n] > M:
        M = tmp[n]
print((tot/N)/M*100)

File: 백준/Bronze/1546. 평균/평균.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        String[] input = scanner.nextLine().split(" ");
        int M = -1;
        double answer = 0;
        for (int i = 0; i < n; i++) {
            int tmp = Integer.parseInt(input[i]);
            answer += tmp;
            if (M < tmp){
                M = tmp;
            }
        }
        answer = answer / n * 100 / M;
        System.out.println(answer);
    }
}
